config/routes.rb

```diff
@@ -41,5 +41,6 @@
 
   namespace :merchant do
     get '/', to: 'dashboard#index', as: 'dashboard'
```

Comment:
I am working on this route for issue #14 and I was wondering how the group felt about us changing this to a #show action instead of an #index action? Technically the user story is saying the path will take the merchant user to their merchant dashboard show page. If we changed the route, we would also need to change the view from index.html.erb to show.html.erb.

Reply:
That makes sense to me. There is only one dashboard per user after all.